### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by FrancescoCeruti**
Enjoy!

### CODE REVIEW

**Comment by trebmuh** [comment]
Checked the French translation, all good.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 1]
```diff
@@ -70,15 +70,15 @@
     <message>
       <location filename="../../../plugins/gst_backend/settings/alsa_sink.py" line="62"/>
       <source>ALSA device</source>
-      <translation type="unfinished">ALSA device</translation>
+      <translation>Appareil ALSA</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Usually "device" is translated into "périphérique", not "appareil".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 2]
```diff
@@ -94,17 +94,17 @@
     <message>
       <location filename="../../../ui/settings/app_pages/general.py" line="95"/>
       <source>Default layout</source>
-      <translation type="unfinished">Default layout</translation>
+      <translation>Disposition par défaut</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: At another place, "layout" is translated as "agencement". I think "disposition" is better.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@
     <message>
       <location filename="../../../ui/settings/app_pages/general.py" line="107"/>
       <source>Application language (require restart)</source>
-      <translation type="unfinished">Application language (require restart)</translation>
+      <translation>Langue de l'application (nécessite le redémarrage)</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "(nécessite le redémarrage)" -> "(nécessite un redémarrage)"

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 4]
```diff
@@ -130,37 +130,37 @@
...
       <location filename="../../../plugins/gst_backend/settings/audio_dynamic.py" line="57"/>
       <source>Expander</source>
-      <translation type="unfinished">Expander</translation>
+      <translation>Expandeur</translation>
     </message>
     <message>
       <location filename="../../../plugins/gst_backend/settings/audio_dynamic.py" line="68"/>
       <source>Soft Knee</source>
-      <translation type="unfinished">Soft Knee</translation>
+      <translation>Pente légère</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "pente", but "coude" ou "virage".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 5]
```diff
@@ -130,37 +130,37 @@
...
     </message>
     <message>
       <location filename="../../../plugins/gst_backend/settings/audio_dynamic.py" line="71"/>
       <source>Hard Knee</source>
-      <translation type="unfinished">Hard Knee</translation>
+      <translation>Pente dure</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: idem

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 6]
```diff
@@ -216,17 +216,17 @@
...
       <location filename="../../../plugins/cart_layout/layout.py" line="193"/>
       <source>Show seek-bars</source>
-      <translation type="unfinished">Show seek-bars</translation>
+      <translation>Afficher les barres de recherche</translation>
     </message>
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="194"/>
       <source>Show dB-meters</source>
-      <translation type="unfinished">Show dB-meters</translation>
+      <translation>Afficher le niveau de décibels</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: There is (at least) another translation of the exact same sentence above. I very much prefer this one.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 7]
```diff
@@ -301,12 +301,12 @@
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="482"/>
       <source>Warning</source>
-      <translation type="unfinished">Warning</translation>
+      <translation>Avertissement</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: There is (at least) another translation of the exact same sentence above. I very much prefer this one.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 8]
```diff
@@ -347,22 +347,22 @@
...
       <location filename="../../../plugins/action_cues/command_cue.py" line="131"/>
       <source>Command to execute, as in a shell</source>
-      <translation type="unfinished">Command to execute, as in a shell</translation>
+      <translation>Commande à exécuter, comme dans un terminal</translation>
     </message>
     <message>
       <location filename="../../../plugins/action_cues/command_cue.py" line="134"/>
       <source>Discard command output</source>
-      <translation type="unfinished">Discard command output</translation>
+      <translation>Annuler la sortie de commande</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "annuler" but "abandonner".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 9]
```diff
@@ -399,145 +399,145 @@
...
       <source>Types</source>
-      <translation type="unfinished">Types</translation>
+      <translation>Types</translation>
     </message>
     <message>
-      <location filename="../../../plugins/controller/protocols/osc.py" line="192"/>
+      <location filename="../../../plugins/controller/protocols/osc.py" line="193"/>
       <source>Arguments</source>
-      <translation type="unfinished">Arguments</translation>
+      <translation>Paramètres</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Note "paramètres" but "arguments".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 10]
```diff
@@ -813,12 +800,12 @@
...
       <location filename="../../../ui/settings/app_pages/cue.py" line="53"/>
       <source>Interrupt fade</source>
-      <translation type="unfinished">Interrupt fade</translation>
+      <translation>Interrompre le fondu</translation>
     </message>
     <message>
       <location filename="../../../ui/settings/app_pages/cue.py" line="54"/>
       <source>Fade actions</source>
-      <translation type="unfinished">Fade actions</translation>
+      <translation>Action de fondu</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "Action" but "Actions" (plural form).

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 11]
```diff
@@ -1145,78 +1140,63 @@
...
+      <location filename="../../../plugins/list_layout/layout.py" line="42"/>
       <source>List Layout</source>
-      <translation type="unfinished">List Layout</translation>
+      <translation>Disposition en liste</translation>
     </message>
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="44"/>
       <source>Cart Layout</source>
-      <translation type="unfinished">Cart Layout</translation>
+      <translation>Disposition en carte</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "cart" -> "carte" ??? Shoudn't that be "panier" or "charriot"?

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 12]
```diff
@@ -1415,17 +1395,17 @@
...
       <location filename="../../../ui/logging/dialog.py" line="86"/>
       <source>Show details</source>
       <translation>Afficher les détails</translation>
     </message>
     <message>
-      <location filename="../../../ui/logging/viewer.py" line="55"/>
+      <location filename="../../../ui/logging/viewer.py" line="53"/>
       <source>Linux Show Player - Log Viewer</source>
-      <translation type="unfinished">Linux Show Player - Log Viewer</translation>
+      <translation>Lecteur Linux Show - Visionneur de log</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: We should keep the name "Linux Audio Player" as a whole, and not translate it.
"log" -> "journal"

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 13]
```diff
@@ -1450,7 +1430,7 @@
     <message>
       <location filename="../../../ui/logging/common.py" line="39"/>
       <source>Logger name</source>
-      <translation type="unfinished">Logger name</translation>
+      <translation>Nom de log</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "log" -> "journaliseur"

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 14]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../ui/logging/common.py" line="49"/>
       <source>Thread ID</source>
-      <translation type="unfinished">Thread ID</translation>
+      <translation>ID du fil</translation>
     </message>
     <message>
       <location filename="../../../ui/logging/common.py" line="50"/>
       <source>Thread name</source>
-      <translation type="unfinished">Thread name</translation>
+      <translation>Nom du sujet</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: That isn't "sujet"!!!, that is "fil". But we might be better here to keep "thread" even.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 15]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="67"/>
       <source>Channel</source>
-      <translation type="unfinished">Channel</translation>
+      <translation>Canal</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="68"/>
       <source>Note</source>
-      <translation type="unfinished">Note</translation>
+      <translation>Commentaire</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Sure it is "note" as a "comment" here and not as a musical "note"?

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 16]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="72"/>
       <source>Value</source>
-      <translation type="unfinished">Value</translation>
+      <translation>Valeur</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="73"/>
       <source>Song</source>
-      <translation type="unfinished">Song</translation>
+      <translation>Musique</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "musique" but "morceau"!!

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 17]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="51"/>
       <source>Note OFF</source>
-      <translation type="unfinished">Note OFF</translation>
+      <translation>Note OFF</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="52"/>
       <source>Polyphonic After-touch</source>
-      <translation type="unfinished">Polyphonic After-touch</translation>
+      <translation>Post-traitement polyphonique</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: No "post-traitement"!!! Keep it as "after-touch", it's a well known MIDI feature.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 18]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="53"/>
       <source>Control/Mode Change</source>
-      <translation type="unfinished">Control/Mode Change</translation>
+      <translation>Changement de contrôle/mode</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="56"/>
       <source>Program Change</source>
-      <translation type="unfinished">Program Change</translation>
+      <translation>Programme de changement</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: No "Programme de changement"!!! Keep it as "program change", it's a well known MIDI feature.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 19]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="52"/>
       <source>Polyphonic After-touch</source>
-      <translation type="unfinished">Polyphonic After-touch</translation>
+      <translation>Post-traitement polyphonique</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="53"/>
       <source>Control/Mode Change</source>
-      <translation type="unfinished">Control/Mode Change</translation>
+      <translation>Changement de contrôle/mode</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: No "Changement de contrôle/mode"!!! Keep it as "Control/Mode Change", it's a well known MIDI feature.

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 20]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="56"/>
       <source>Program Change</source>
-      <translation type="unfinished">Program Change</translation>
+      <translation>Programme de changement</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="57"/>
       <source>Channel After-touch</source>
-      <translation type="unfinished">Channel After-touch</translation>
+      <translation>Canal après-toucher</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: _Same here, well know MIDI feature, keep it in English._

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 21]
```diff
@@ -1500,323 +1480,313 @@
...
       <location filename="../../../plugins/midi/midi_utils.py" line="58"/>
       <source>Pitch Bend Change</source>
-      <translation type="unfinished">Pitch Bend Change</translation>
+      <translation>Changement de courbe du pitch</translation>
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="59"/>
       <source>Song Select</source>
-      <translation type="unfinished">Song Select</translation>
+      <translation>Sélection de musique</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "musique" but "morceau".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 22]
```diff
@@ -1500,323 +1480,313 @@
...
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_utils.py" line="60"/>
       <source>Song Position</source>
-      <translation type="unfinished">Song Position</translation>
+      <translation>Position de musique</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Still not "musique" but "morceau".

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 23]
```diff
@@ -1852,370 +1822,370 @@
...
       <source>Type</source>
-      <translation type="unfinished">Type</translation>
+      <translation>Type</translation>
     </message>
     <message>
-      <location filename="../../../plugins/controller/protocols/osc.py" line="73"/>
+      <location filename="../../../plugins/controller/protocols/osc.py" line="74"/>
       <source>Argument</source>
-      <translation type="unfinished">Argument</translation>
+      <translation>Paramètre</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "paramètre" but "argument" ("paramètre" in FR is "setting" in EN).

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 24]
```diff
@@ -1852,370 +1822,370 @@
...
       <location filename="../../../plugins/__init__.py" line="63"/>
       <source>Failed to load &quot;{}&quot;</source>
-      <translation type="unfinished">Failed to load &quot;{}&quot;</translation>
+      <translation>Échec du chargement de &quot;{}&quot;</translation>
     </message>
     <message>
       <location filename="../../../plugins/__init__.py" line="160"/>
       <source>Failed to terminate plugin: &quot;{}&quot;</source>
-      <translation type="unfinished">Failed to terminate plugin: &quot;{}&quot;</translation>
+      <translation>Impossible de mettre fin au plugin : &quot;{}&quot;</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "plugin" -> "greffon"

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 25]
```diff
@@ -1852,370 +1822,370 @@
...
+      <translation>Pré-réglages</translation>
     </message>
   </context>
   <context>
     <name>Presets</name>
     <message>
       <location filename="../../../plugins/presets/presets_ui.py" line="67"/>
       <source>Cannot scan presets</source>
-      <translation type="unfinished">Cannot scan presets</translation>
+      <translation>Impossible de charger les prés-réglages</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "pré-réglages" (not "prés-réglages")

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 26]
```diff
@@ -2321,7 +2291,7 @@
   <context>
     <name>QColorButton</name>
     <message>
-      <location filename="../../../ui/widgets/qcolorbutton.py" line="38"/>
+      <location filename="../../../ui/widgets/colorbutton.py" line="38"/>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: mmm well... @FrancescoCeruti it looks like a typo here. A translator shouldn't change the code, isn't?

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 27]
```diff
@@ -2537,228 +2507,228 @@
...
       <location filename="../../../plugins/action_cues/collection_cue.py" line="60"/>
       <source>Edit Collection</source>
-      <translation type="unfinished">Edit Collection</translation>
+      <translation>Modifier la collection</translation>
     </message>
     <message>
       <location filename="../../../plugins/action_cues/stop_all.py" line="46"/>
       <source>Stop Settings</source>
-      <translation type="unfinished">Stop Settings</translation>
+      <translation>Paramètre d'arrêt</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: "Paramètres" (plural form)

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 28]
```diff
@@ -2537,228 +2507,228 @@
...
       <source>OSC Controls</source>
-      <translation type="unfinished">OSC Controls</translation>
+      <translation>Contrôles OSC</translation>
     </message>
     <message>
-      <location filename="../../../plugins/midi/midi_cue.py" line="52"/>
+      <location filename="../../../plugins/midi/midi_cue.py" line="53"/>
       <source>MIDI Settings</source>
-      <translation type="unfinished">MIDI Settings</translation>
+      <translation>Préférences MIDI</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: Not "préférences" but "paramètres"

File: lisp/i18n/ts/fr_FR/lisp.ts [hunk 29]
```diff
@@ -2537,228 +2507,228 @@
...
     </message>
     <message>
       <location filename="../../../plugins/midi/midi_settings.py" line="33"/>
       <source>MIDI settings</source>
-      <translation type="unfinished">MIDI settings</translation>
+      <translation>Préférences MIDI</translation>
```
Comment (trebmuh) [inline lisp/i18n/ts/fr_FR/lisp.ts]: _same here_

**Comment by trebmuh** [comment]
@FrancescoCeruti : wipe out my last "review" here please, there are a bunch of errors/typos/...
I've started to make a correct review but stopped because I'm not sure anyone will read those. Am I right to assume that the translation is happenning on Weblate/Transifex/...externally? Please advise.

**Comment by FrancescoCeruti** [comment]
@trebmuh yes, the "New Crowdin translations" pull-request are coming from [crowdin](https://crowdin.com/project/linux-show-player). If you want you can login there with the github account and comment/change directly there :-)

**Pull request merged**